build-system/tasks/issues-tasks.js

```diff
@@ -0,0 +1,275 @@
+                    // 20 is the number for Milestone 'Pending Triage' 
+                    updates.push(applyMilestone(issue, issueNewMilestone));
+                  }
+                }
+              }
+            }
+          }
+        } else {
+          if (milestone == null) {
+            // 20 is the number for Milestone 'Pending Triage' 
+            updates.push(applyMilestone(issue, issueNewMilestone));     
+          } else {
+            if (milestoneTitle == 'Prioritized FRs' || milestoneTitle == 'New FRs') {
+              updates.push(applyLabel(issue, 'Type: Feature Request'));
+            } else {
+              if (milestoneTitle == 'Backlog Bugs' || milestoneTitle.startsWith('Sprint')) {
+                updates.push(applyLabel(issue, 'Type: Bug'));
+              }
+            } 
+          }                              
+        }
+        // Apply default priority if no priority
+        if (hasPriority == false && milestoneTitle != 'New FRs' && 
+          milestoneTitle != '3P Implementation' && milestoneTitle != 'Pending Triage' 
+          && milestone != null) { 
+          updates.push(applyLabel(issue, 'P2: Soon')); 
+        }
+        return Promise.all(updates);
+      });
+    });
+    assert(Array.isArray(issues), `issues must be an array.`);
+    return issues;
+  });	
+}
+
+function errHandler(err) {
+  var msg = err;
+  if (err.message) {
+    msg = err.message;
+  }
+  util.log(util.colors.red(msg));
+}
+
+function applyMilestone(issue, milestoneNumber) {
+  var options = extend({}, optionsMilestone);
+  options.qs = {
+    state: 'open',
+    per_page: 1000,
+    access_token: GITHUB_ACCESS_TOKEN,
+  };
+
+  issue.milestone = milestoneNumber;
+  return githubRequest(
+      '/issues/' + issue.number,
+      'PATCH',
+      issue.milestone, 'milestone').then(function() {
+        if (verbose) {
+          util.log(util.colors.green(
+              'Milestone applied ' + milestone + ' for #' + issue.number));
+        }
+      });
+}
+
+function applyLabel(issue, label) {
+  var options = extend({}, optionsMilestone);
+  options.qs = {
+    state: 'open',
+    per_page: 1000,
+    access_token: GITHUB_ACCESS_TOKEN,
+  };
+  return githubRequest(
+      '/issues/' + issue.number + '/labels',
+      'POST',
+      [label], 'label').then(function() {
+        if (verbose) {
+          util.log(util.colors.green(
+              'Label applied ' + label + ' for #' + issue.number));
+        }
+      });
+}
+
+/**
+ * @param {string} path
+ * @param {string=} opt_method
+ * @param {*} opt_data
+ * @param {string} typeRequest
+ * @return {!Promise<*>}
+ */
+function githubRequest(path, opt_method, opt_data, typeRequest) {
+  var options = {
+    // Url to repository 
+    url: 'https://api.github.com/repos/ampproject/amphtml' + path,
+    body: {},
+    headers: {
+      'User-Agent': 'amp-changelog-gulp-task',
+      'Accept': 'application/vnd.github.v3+json'
+    },
+    qs: {
```

Comment:
What is `qs`? If it isn't part of the github API spec, I suggest using a more descriptive (but short) name.

Reply:
I tried to be consistent with what was used in other files, so I'll prefer to keep it with the same name.